systemvm/patches/debian/config/opt/cloud/bin/merge.py

```diff
@@ -272,5 +275,10 @@ def setPath(self, path):
     def __moveFile(self, origPath, path):
         if not os.path.exists(path):
             os.makedirs(path)
-        timestamp = str(int(round(time.time())))
-        os.rename(origPath, path + "/" + self.fileName + "." + timestamp)
+
+        zipped_file_name = path + "/" + self.fileName + "." + str(uuid.uuid4()) + ".gz"
```

Comment:
would it make sense to have the timestamp in the filename as well? for easy ordering